apps/platform-app/src/app/[locale]/(profile)/_actions/queries/findProfileByLinkedinUsername.ts

```diff
@@ -0,0 +1,18 @@
+'use server'
+
+import { createProfileModel } from '@/app/[locale]/(profile)/_models/profile.model'
+import { getProfileByLinkedinUsername } from '@/backend/profile/profile.service'
+import { withSentry } from '@/utils/errHandling'
+
+export const findProfileByLinkedinUsername = withSentry(
+  async (username: string) => {
+    const profile = await getProfileByLinkedinUsername(username)
+
+    if (!profile) {
+      //   throw new Error(`Profile with username ${username} not found`)
```

Comment:
**suggestion**: Consider logging the error message before returning null for better traceability in case of issues.